sources/Afup/Comptabilite/Facture.php

```diff
@@ -9,6 +9,9 @@
 
 class Facture
 {
+    private $cipher  = 'aes-256-gcm';
```

Comment:
L'algo `aes-256-gcm` demande l'ajout de l'argument `tag` pour le chiffrement et déchiffrement. Pour éviter sont utilisation, il est possible d'utiliser `aes-256-cbc`.

```suggestion
    private $cipher  = 'aes-256-cbc';
```